example/babel.config.js

```diff
@@ -1,3 +1,13 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
+    [
+      'module-resolver',
+      {
+        alias: {
+          'react-native-toast-banner': '..',
```

Comment:
Try `../index`

Reply:
I am suspecting `package.json` might disturb this behavior since it is directing to built one.

Reply:
😭 no luck. I just tried that one and such like `../lib/index.js` but I get the same error.